activerecord/lib/active_record/errors.rb

```diff
@@ -552,6 +552,11 @@ class SerializationFailure < TransactionRollbackError
   class Deadlocked < TransactionRollbackError
   end
 
+  # OrderError is raised when a relation needs to be ordered but does not have any
+  # order columns to use.
+  class OrderError < ActiveRecordError
+  end
+
   # IrreversibleOrderError is raised when a relation's order is too complex for
   # +reverse_order+ to automatically reverse.
   class IrreversibleOrderError < ActiveRecordError
```

Comment:
I like how `IrreversibleOrderError` is self-documenting; it has a sentence describing it but you can generally tell what it does without having to look it up. Should `OrderError` be something like `MissingRequiredOrderError` so that its more self-documenting?

Reply:
I like that, will action 👍🏽

Reply:
Done!